src/Components/Facility/models.tsx

```diff
@@ -222,6 +223,15 @@ export interface CurrentBed {
   meta: Record<string, any>;
 }
 
+export interface ConsultationBedRequest {
+  bed?: string;
+  consultation: string;
+  limit?: number;
+  offset?: number;
+}
+
+export type ConsultationBedResponse = PaginatedResponse<CurrentBed>;
```

Comment:
You can wrap it `PaginatedResponse` in `api.tsx` file itself instead of creating a new type right?

Reply:
(Just to keep it consistent on how other api routes are defined. It would be easier to look and know that this route is a paginated route.)